src/cst1201/Simulator.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cst1201;
+
+/**
+ *
+ * @author Mingjiehuang
+ */
+public class Simulator {
+
+    public static void main(String[] args) {
+
+        HouseDrawing turtle = new HouseDrawing();

```

Comment:
Weird name for a `HouseDrawing` reference variable. Perhaps `drawing` would be more intuitive.